src/components/application_manager/include/application_manager/resumption/resumption_data.h

```diff
@@ -52,7 +56,8 @@ class ResumptionData {
   /**
    * @brief Constructor of ResumptionData
    */
-  ResumptionData();
+  ResumptionData(
+      const application_manager::ApplicationManagerSettings& settings);
```

Comment:
@VVeremjova , no i don`t. Init if abstract method and must be overridden in child-class.